Komet/Preferences.xib

```diff
@@ -75,13 +76,13 @@
             <connections>
                 <outlet property="delegate" destination="-2" id="TCu-6C-Rbz"/>
             </connections>
-            <point key="canvasLocation" x="270" y="-80.5"/>
+            <point key="canvasLocation" x="129" y="-135"/>
         </window>
         <customView id="h9g-vJ-Iwb" userLabel="Fonts View">
             <rect key="frame" x="0.0" y="0.0" width="268" height="139"/>
             <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
             <subviews>
-                <textField verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" verticalCompressionResistancePriority="800" translatesAutoresizingMaskIntoConstraints="NO" id="P9V-1A-nFH">
```

Comment:
This deluge of `allowsCharacterPickerTouchBarItem="NO"` additions seems to have been done automatically by XCode [after updating macOS to 20.12](https://developer.apple.com/library/content/releasenotes/AppKit/RN-AppKit/).